File: Sirius/Entities/Actor.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Sirius.Entities
{
    public class Actor
    {
        public int ID { get; set; }
        public String Name { get; set; }
        public String Birthplace { get; set; }
        public String Birthday { get; set; }
        public String Biography { get; set; }

        public DateTime getBirthday()
        {
            if(this.Birthday == null) return new DateTime();

            long timestamp = Int64.Parse(this.Birthday);
            DateTime startDateTime = new DateTime(1970, 1, 1, 0, 0, 0, 0);
            return startDateTime.AddMilliseconds(timestamp).ToLocalTime();
        }

        public List<Series> filmography { get; set; }

        public Role playedIn(Series movie, String role)
        {
            return null;
        }
    }
}

File: SiriusFE/frontend/static/js/models/Actor.js

export class Actor {
    constructor(id, name, sex, birthplace, birthday, biography) {
        this.id = id;
        this.name = name;
        this.sex=sex;
        this.birthplace = birthplace;
        this.birthday = birthday;
        this.biography = biography;
    }

    print() {
        console.log(this.id + "\n"+this.name+ "\n"+this.birthplace + "\n"+this.birthday + "\n"+this.biography + "\n");
    }
}